src/test/resources/fixtures/govuk-frontend/test-fixtures/summary-list-with-actions/output.txt

```diff
@@ -52,18 +45,11 @@
         </p>
         </dd>
           <dd class="govuk-summary-list__actions">
-              <ul class="govuk-summary-list__actions-list">
-                  <li class="govuk-summary-list__actions-list-item">
-                    <a class="govuk-link" href="#">
+              <ul class="govuk-summary-list__actions-list"><li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">
```

Comment:
@kyle-bowden I think for this we need to update the summary list and list with actions components so that the twirl components don't have whitespace, because I think whitespace in the html will impact how it is rendered